### ISSUE TITLE
Неправильная работа Имплантов Лояльности

### ISSUE THREAD

**Issue by Motzord** [GammaStation/Gamma-Station#498]
Значит так, судя по педивикии гаммы https://wiki.taucetistation.org/Security_Items#.D0.98.D0.BC.D0.BF.D0.BB.D0.B0.D0.BD.D1.82_.D0.BB.D0.BE.D1.8F.D0.BB.D1.8C.D0.BD.D0.BE.D1.81.D1.82.D0.B8_.28Loyalty_Implant.29

Импланты лояльности работают лишь в случае ревы, но ими в данный момент можно пичкать кого угодно включая оперативников, триторов, генокрадов и прочих веселых парней.
>Как повторить? Берете и пичкаете заведомо известного тритора/оперативника имплантом лояльности.
>Примечания: Из-за этого был слит раунд.

**Comment by RichardJones1** [comment]
Лол. Так в чем проблема?)
Ролька спалилась, о нет
Ну или пгшер-офицер

**Comment by LudwigVonChesterfield** [comment]
Так-то всё правильно. Он говорит что импланты не должны колоться кому-либо кроме реверов.

**Comment by Miracler112** [comment]
Тем не менее имплант - промывка мозгов и возможность его колоть кому-либо стало фичей, но если оно так вообще не задумывалось, в чем я сомневаюсь, то ладно, поправят как-нибудь

**Comment by LudwigVonChesterfield** [comment]
Либо вики менять - либо код.

**Comment by RichardJones1** [comment]
А почему импланты лояльности не должны колоться  врагам нт?
Хотя колоть без веской причины говно.
Но вы сами выпилили комнату казни)

**Comment by omejka** [comment]
Два стула
Либо мы вкалываем микрочелу имплант и отпускаем
Либо мы кидаем его в карцер\хуярим на месте леталом
